### PR TITLE
Fixes Issue #584

### PR DESCRIPTION
**Pull request by jrrickard**
Adds RootDir property to GraphData.

 This was seen on a CentOS system without aufs support.

---
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/585)

### CODE REVIEW

**Comment by jrrickard** [comment]
Ooops, sorry. Looks like I forgot to turn off my team's code formatter. Let me swap and resubmit

File: src/main/java/com/github/dockerjava/api/command/GraphData.java [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import javax.annotation.CheckForNull;
+
 import org.apache.commons.lang.builder.EqualsBuilder;
 import org.apache.commons.lang.builder.HashCodeBuilder;
 import org.apache.commons.lang.builder.ToStringBuilder;
 
-import javax.annotation.CheckForNull;
+import com.fasterxml.jackson.annotation.JsonProperty;
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/GraphData.java]: please don't reorder imports

**Comment by KostyaSha** [comment]
Please remove all coding style changes.

**Comment by KostyaSha** [comment]
IDEA default or eclipse would mostly match, for eclipse some config file even committed in repository. For idea i still had no time, sorry for it. Formatting really makes difficult reviews.

File: src/main/java/com/github/dockerjava/api/command/GraphData.java (hunk 1 above)
Comment (jrrickard) [inline src/main/java/com/github/dockerjava/api/command/GraphData.java]: yeah sorry, that was my other code formatter.

**Comment by jrrickard** [comment]
@KostyaSha yeah sorry, going to reformat and resubmit. I found the code style under the /etc directory. Reformatting with that.

**Comment by KostyaSha** [comment]
test it 1.11

**Comment by KostyaSha** [comment]
test it-1.11

File: src/main/java/com/github/dockerjava/api/command/GraphData.java [hunk 2]
```diff
@@ -22,6 +26,22 @@
...
     /**
+     * @see #rootDir
+     */
+    @CheckForNull
+    public String getRootDir() {
+        return rootDir;
+    }
+
+    /**
+     * @see #deviceId
```
Comment (marcuslinke) [inline src/main/java/com/github/dockerjava/api/command/GraphData.java]: Please fix comment. Thanks.

**Comment by KostyaSha** [comment]
Now looks good :) Please ignore commit status, it was PR builder against docker-beta that i started configuring :)

File: src/main/java/com/github/dockerjava/api/command/GraphData.java (hunk 2 above)
Comment (jrrickard) [inline src/main/java/com/github/dockerjava/api/command/GraphData.java]: Oops, just amended to fix that. Sorry!

**Pull request merged**